wolfcrypt/src/wc_port.c

```diff
@@ -348,6 +348,59 @@ wolfSSL_Mutex* wc_InitAndAllocMutex(void)
     return m;
 }
 
+#ifdef USE_WOLF_STRTOK
+/* String token (delim) search. If str is null use nextp. */
+char* wc_strtok(char *str, const char *delim, char **nextp)
```

Comment:
Why is this function not thread safe?  I see no use of globals or statics on a first read.